config/database.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+
+dotenv.config();
+const { MONGO_URI } = process.env;
+
+export const connect = () => {
+  mongoose
+    .connect(MONGO_URI, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    })
+    .then(() => {
+      console.log("Successfully connected to database");
```

Comment:
Console log is not a good approach to log information on the server.

The better approach here is to use logger `winston`

Reply:
Thanks for the comment, took note, surely will study it in near future